src/components/company/chart.js

```diff
@@ -112,11 +114,11 @@ const tickStyle = {
 
 const DisplayChart = ({ priceFilter, history }) => {
   return (
-    <ResponsiveContainer width="100%" height={500}>
+    <ResponsiveContainer width="99%" height="100%">
```

Comment:
Does this need to be specified here? Perhaps, it should be specified in the css file

Reply:
I believe the style of recharts elements must be specified inline, otherwise it wouldn't be work.